library/src/main/java/com/heinrichreimersoftware/materialintro/slide/SimpleSlide.java

```diff
@@ -519,10 +539,48 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                 descriptionView.setTextColor(textColorSecondary);
             }
 
+            if (getActivity() instanceof SimpleSlideActivity) {
+                ((SimpleSlideActivity)getActivity()).onSlideViewCreated(this, fragment, id);
+            }
+
             return fragment;
         }
 
         @Override
+        public void onDestroyView() {
+            if (getActivity() instanceof SimpleSlideActivity) {
+                int id = getArguments().getInt(ARGUMENT_ID);
+                ((SimpleSlideActivity)getActivity()).onSlideDestroyView(this, getView(), id);
+            }
+            innerLinearLayout = null;
+            titleView = null;
+            descriptionView = null;
+            imageView = null;
+            super.onDestroyView();
+        }
+
+        public LinearLayout getInnerLinearLayout() {
+            return innerLinearLayout;
```

Comment:
That's fine. I just think, the `SimpleSlide` already has all the rotation code in it and can be reused. I can remove the `getInnerLinearLayout`.

Anyways, following works as well for both layouts (landscape and portrait) currently: `LinearLayout ll = (LinearLayout)fragment.getTitleView().getParent();`